ci/tasks/test-stress/setup-env.sh

```
@@ -1,33 +1,31 @@
+set -eu -o pipefail
+
 main() {
-  source $PWD/bosh-dns-release/ci/assets/utils.sh
-  local output_dir="$PWD/updated-bbl-state/"
-  local bbl_state_env_repo_dir=$PWD/bbl-state
-  trap "commit_bbl_state_dir ${bbl_state_env_repo_dir} ${BBL_STATE_DIR} ${output_dir} 'Update bbl state dir'" EXIT
+  source "$PWD/bosh-dns-release/ci/assets/utils.sh"
 
   export TEST_STRESS_ASSETS=$PWD/bosh-dns-release/ci/assets/test-stress
-  export BOSH_DOCKER_CPI_RELEASE_TARBALL="$( echo $PWD/bosh-docker-cpi-release/*.tgz )"
+  export BOSH_DOCKER_CPI_RELEASE_TARBALL="$( echo "$PWD"/bosh-docker-cpi-release/*.tgz )"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)